src/server/api/controllers/users.controller.js

```diff
@@ -7,22 +7,37 @@ const getUsers = async () => {
 
 const getUsersById = async (id) => {
   if (!id) {
-    throw new HttpError('Bad request. User ID must be an integer and larger than 0', 400);
+    throw new HttpError(
+      'Bad request. User ID must be an integer and larger than 0',
+      400,
+    );
   }
 
   try {
     const users = await knex('users').where({ id });
     if (users.length === 0) {
-      throw new Error(`A user with the specified ID was not found : ${id}`, 404);
+      throw new Error(
+        `A user with the specified ID was not found : ${id}`,
+        404,
+      );
     }
     return users;
   } catch (error) {
     return error.message;
   }
 };
 
+const saveUser = async (data) => {
+  await knex('users').insert(data);
+};
+
+const updateUser = async (idValue, data) => {
```

Comment:
Why did you also add update here? i think thi issue only neeed to POST user

Reply:
I have removed the Update, thanks